client/blocks/signup-form/social.jsx

```diff
@@ -75,7 +74,12 @@ class SocialSignupForm extends Component {
 						responseHandler={ this.handleGoogleResponse }
 						redirectUri={ redirectUri }
 						uxMode={ uxMode }
-						onClick={ this.trackGoogleLogin }
+						onClick={ this.trackSocialLogin.bind( null, 'google' ) }
+					/>
+					<AppleLoginButton
+						clientId={ config( 'apple_oauth_client_id' ) }
+						redirectUri={ `https://${ window.location.host }/start` }
```

Comment:
Let's not use `redirectUri` for now, and only support the popup flow.
Should be pretty close to https://github.com/Automattic/wp-calypso/pull/35428/files#diff-03f7d5aab5df5cd3f255095d092ff252R109

Reply:
Shouldn't we hit the [`handleSocialResponse()` method](https://github.com/Automattic/wp-calypso/blob/19957a61f68c2a2ce24bdd25b0bfa855b6fbc215/client/signup/steps/user/index.jsx#L205) at some point?